src/main/resources/application.properties

```diff
@@ -1,2 +1,3 @@
 quarkus.kogito.devservices.enabled=false
-quarkus.devservices.enabled=false
\ No newline at end of file
+quarkus.devservices.enabled=false
+quarkus.native.resources.includes=**swidtag
```

Comment:
@martinweiler ohh wait, no - I don't think this is needed. In case of native build, the library will write to the file system a new swidtag - different from the ILMT library :) 

@thiagoelg FYI ^

Reply:
Without that property, the app will not be able to load ibm.com_IBM_Process_Automation_Manager_Open_Edition.swidtag included in the ilmt library and will fail:
```
2023-06-21 14:27:57,219 ERROR [io.qua.run.Application] (main) Failed to start application (with profile [prod]): java.lang.NullPointerException
	at java.base@17.0.7/java.io.Reader.<init>(Reader.java:168)
	at java.base@17.0.7/java.io.InputStreamReader.<init>(InputStreamReader.java:76)
	at com.ibm.bamoe.ilmt.common.SwidFileGenerator.getSwidContent(SwidFileGenerator.java:46)
	at com.ibm.bamoe.ilmt.common.SwidFileGenerator.createSwidFile(SwidFileGenerator.java:28)
```

Reply:
Without this property a new .swidtag was created on the first run, but I got no errors

Reply:
No errors after running in native mode?
```
$ mvn package -Dnative
$ ./target/code-with-bamoe-1.0.0-SNAPSHOT-runner
```

Reply:
Oh, I haven't tried with that flag. (I don't even have the Native Image installed)

Reply:
The issue only shows up with the native image. Maybe we should add it to the template as:
```
# enable this property when creating a native image
# quarkus.native.resources.includes=**swidtag
```